2023-03-11 22:33:55,459 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Configure stats pid to 42889
2023-03-11 22:33:55,459 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Loading settings from /home/mikhail/.config/wandb/settings
2023-03-11 22:33:55,459 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Loading settings from /home/mikhail/github/Facades/wandb/settings
2023-03-11 22:33:55,459 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-11 22:33:55,459 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-11 22:33:55,459 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Applying login settings: {'anonymous': 'must'}
2023-03-11 22:33:55,459 INFO    MainThread:42889 [wandb_init.py:_log_setup():476] Logging user logs to /home/mikhail/github/Facades/wandb/run-20230311_223355-38yh7j1q/logs/debug.log
2023-03-11 22:33:55,460 INFO    MainThread:42889 [wandb_init.py:_log_setup():477] Logging internal logs to /home/mikhail/github/Facades/wandb/run-20230311_223355-38yh7j1q/logs/debug-internal.log
2023-03-11 22:33:55,460 INFO    MainThread:42889 [wandb_init.py:init():516] calling init triggers
2023-03-11 22:33:55,460 INFO    MainThread:42889 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-03-11 22:33:55,460 INFO    MainThread:42889 [wandb_init.py:init():535] re-initializing run, found existing run on stack: 2pv760p2
2023-03-11 22:33:55,468 INFO    MainThread:42889 [wandb_run.py:_finish():1746] finishing run anony-moose-512331/U-Net/2pv760p2
2023-03-11 22:33:55,469 ERROR   MainThread:42889 [jupyter.py:save_history():440] Run pip install nbformat to save notebook history
2023-03-11 22:33:55,469 INFO    MainThread:42889 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2023-03-11 22:33:55,469 INFO    MainThread:42889 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2023-03-11 22:33:55,469 INFO    MainThread:42889 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2023-03-11 22:33:55,470 INFO    MainThread:42889 [wandb_run.py:_restore():1932] restore
2023-03-11 22:33:55,470 INFO    MainThread:42889 [wandb_run.py:_restore():1938] restore done
2023-03-11 22:34:03,032 INFO    MainThread:42889 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2023-03-11 22:34:03,032 INFO    MainThread:42889 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2023-03-11 22:34:03,042 INFO    MainThread:42889 [wandb_run.py:_footer_sync_info():3333] logging synced files
2023-03-11 22:34:03,051 INFO    MainThread:42889 [wandb_init.py:init():569] starting backend
2023-03-11 22:34:03,052 INFO    MainThread:42889 [wandb_init.py:init():573] setting up manager
2023-03-11 22:34:03,055 INFO    MainThread:42889 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-11 22:34:03,057 INFO    MainThread:42889 [wandb_init.py:init():580] backend started and connected
2023-03-11 22:34:03,063 INFO    MainThread:42889 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2023-03-11 22:34:03,063 INFO    MainThread:42889 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-11 22:34:03,063 INFO    MainThread:42889 [wandb_init.py:init():658] updated telemetry
2023-03-11 22:34:03,064 INFO    MainThread:42889 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2023-03-11 22:34:05,509 INFO    MainThread:42889 [wandb_run.py:_on_init():2000] communicating current version
2023-03-11 22:34:05,940 INFO    MainThread:42889 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.13.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-11 22:34:05,940 INFO    MainThread:42889 [wandb_init.py:init():728] starting run threads in backend
2023-03-11 22:34:05,988 INFO    MainThread:42889 [wandb_run.py:_console_start():1980] atexit reg
2023-03-11 22:34:05,989 INFO    MainThread:42889 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2023-03-11 22:34:05,989 INFO    MainThread:42889 [wandb_run.py:_redirect():1903] Wrapping output streams.
2023-03-11 22:34:05,989 INFO    MainThread:42889 [wandb_run.py:_redirect():1925] Redirects installed.
2023-03-11 22:34:05,990 INFO    MainThread:42889 [wandb_init.py:init():765] run started, returning control to user process
2023-03-11 22:34:15,180 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Configure stats pid to 42889
2023-03-11 22:34:15,180 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Loading settings from /home/mikhail/.config/wandb/settings
2023-03-11 22:34:15,180 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Loading settings from /home/mikhail/github/Facades/wandb/settings
2023-03-11 22:34:15,180 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-11 22:34:15,180 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-11 22:34:15,180 INFO    MainThread:42889 [wandb_setup.py:_flush():68] Applying login settings: {'anonymous': 'must'}
2023-03-11 22:34:15,181 INFO    MainThread:42889 [wandb_init.py:_log_setup():476] Logging user logs to /home/mikhail/github/Facades/wandb/run-20230311_223415-1cjzdi8h/logs/debug.log
2023-03-11 22:34:15,181 INFO    MainThread:42889 [wandb_init.py:_log_setup():477] Logging internal logs to /home/mikhail/github/Facades/wandb/run-20230311_223415-1cjzdi8h/logs/debug-internal.log
2023-03-11 22:34:15,181 INFO    MainThread:42889 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f819828c130>
2023-03-11 22:34:15,182 INFO    MainThread:42889 [wandb_init.py:init():516] calling init triggers
2023-03-11 22:34:15,182 INFO    MainThread:42889 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-03-11 22:34:15,182 INFO    MainThread:42889 [wandb_init.py:init():535] re-initializing run, found existing run on stack: 38yh7j1q
2023-03-11 22:34:15,186 INFO    MainThread:42889 [wandb_run.py:_finish():1746] finishing run anony-moose-512331/U-Net/38yh7j1q
2023-03-11 22:34:15,187 INFO    MainThread:42889 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2023-03-11 22:34:15,188 INFO    MainThread:42889 [wandb_run.py:_restore():1932] restore
2023-03-11 22:34:15,189 INFO    MainThread:42889 [wandb_run.py:_restore():1938] restore done
2023-03-11 22:34:22,285 INFO    MainThread:42889 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2023-03-11 22:34:22,286 INFO    MainThread:42889 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2023-03-11 22:34:22,296 INFO    MainThread:42889 [wandb_run.py:_footer_sync_info():3333] logging synced files
